Assets/Treasure/TDK/Runtime/API/Transaction.cs

```diff
@@ -27,7 +27,7 @@ public struct TransactionOverrides
         }
 
         public string address;
-        public object[] abi;
+        public string abi;
```

Comment:
This might be an annoying breaking change for Zeeverse. Can it support both and encode to `string` within the TDK if an `object[]` is provided? See JS SDK example for the other direction: https://github.com/TreasureProject/tdk-js/blob/main/apps/api/src/routes/transactions.ts#L92

Reply:
Hmm, i'll look into it, i assume i'd need to keep an `object` typing since that would allow for both `object[]` and `string`? its some advance c# typing but that sounds like it could work.

And in the case its an `object[]` we should parse it into a string somehow, but we dont know its actual type. Perhaps i call `JsonConvert.SerializeObject` and 🙏

Reply:
Perhaps we could ask how they pass the abis so i can test it at least

Reply:
Asked zeeverse, it seems they already have a `string`, and are having to convert it to fit the `object[]` type:
![image](https://github.com/user-attachments/assets/4cb3ae13-1690-4fef-9942-449e839729a0)

I think it makes sense to keep it as string only so they can remove this extra step (and for clarity of use, it makes more sense than to have an `object` type param)